packages/ng/src/app/user/tile/user-tile.component.ts

```diff
@@ -11,16 +11,25 @@ import { LuUserDisplayPipe } from '../display';
 	styleUrls: ['./user-tile.component.scss']
 })
 export class LuUserTileComponent {
-
 	/**
 	 * IUser to display.
 	 */
 	@Input() user: IUser;
 
+	/**
+	 * User Display format.
+	 * It is set to 'lf' by default
+	 */
+	@Input() displayFormat = 'lf';
+
 	/**
 	 * IUser role to display
 	 */
 	@Input() role: string;
 
+	get userName() {
```

Comment:
As it is for internal use, can you please set this method to `private`?

Reply:
Done